cmds/core/watchdogd/watchdogd_linux.go

```diff
@@ -8,6 +8,8 @@
 //
 //	watchdogd run [OPTIONS]
 //	    Run the watchdogd in a child process (does not daemonize).
+//	watchtdog pid [tinygo only]
```

Comment:
sorry, I missed something, why is this tinygo only?

Reply:
The idea was to use the old pid-based version for tinygo only and the new uds-based version for non-tinygo. In theory, they should do the same thing but the uds version seems to be better and more reliable (? @10000TB), but also just pets a watchdog, so no need to have both versions in the non-tinygo build.